File: code/platform/src/main/java/org/fbme/ide/platform/editor/HeaderedEditorDataKeys.java
package org.fbme.ide.platform.editor;

import com.intellij.openapi.actionSystem.CommonDataKeys;
import com.intellij.openapi.actionSystem.DataKey;

public class HeaderedEditorDataKeys extends CommonDataKeys {

    public static final DataKey<HeaderedNodeEditor> EDITOR = DataKey.create("headered-editor");
}

File: code/platform/src/main/kotlin/org/fbme/ide/platform/editor/HeaderedEditorDataKeys.kt
package org.fbme.ide.platform.editor

import com.intellij.openapi.actionSystem.CommonDataKeys
import com.intellij.openapi.actionSystem.DataKey

object HeaderedEditorDataKeys : CommonDataKeys() {
    val EDITOR = DataKey.create<HeaderedNodeEditor?>("headered-editor")
}
